Source/Examples/ExampleLibrary/Axes/AxisExamples.cs

```diff
@@ -1409,5 +1346,81 @@ public static PlotModel MultiVerticalAxes()
 
             return plotModel;
         }
+
```

Comment:
Could we have an example with a vertical axis with rotated text? At the moment, it's hard to visually check that the transformations are doing the right thing.

Reply:
You could always check the transposed version of the example with rotated text.
Or would you prefer to have it explicitly?

Reply:
… no, that's fine. I just forget they were there ;)